automl/google/cloud/automl_v1beta1/tables/gcs_client.py

```diff
@@ -54,14 +54,18 @@ def __init__(self, bucket_name=None, client=None, credentials=None):
                 credentials identify this application to the service. If none
                 are specified, the client will attempt to ascertain the
                 credentials from the environment.
+            project (Optional[string]): The project to attach to the underlying
```

Comment:
Could this be re-worded to mention that this is the project ID?
```suggestion
            project (Optional[str]): The project to attach to the underlying
```

Reply:
Sure. Should I change the wording and string type for the existing `TablesClient` docstrings as well? It would be nice for them to be consistent.

Reply:
Updated docstrings to reference project ID explicitly and changed `string` to `str` in type annotations.